releases/v20.2.0-beta.1.md

```diff
@@ -0,0 +1,333 @@
+- Introduced a new `VIEWACTIVITY` role option. This grants non-admin roles the ability to see other users' sessions and queries through [`SHOW SESSIONS`](../v20.2/show-sessions.html), [`SHOW QUERIES`](../v20.2/show-queries.html), and the [Admin UI Statements page](../v20.2/admin-ui-statements-page.html). [#53291][#53291]
+- Introduced a new `CANCELQUERY` role option. This grants non-admin roles the ability to cancel other users' queries and sessions. Note that non-admins are not allowed to cancel the queries or sessions of admins. [#53291][#53291]
+- Implemented `ST_Expand` for `GEOMETRY`-based types. [#53326][#53326]
+- Implemented the geometry [builtins](../v20.2/functions-and-operators.html#built-in-functions) `ST_Multi`, `ST_CollectionExtract`, and `ST_CollectionHomogenize`. [#53287][#53287]
+- Previously, `selectivity` information would be included in [`EXPLAIN ANALYZE`](../v20.2/explain-analyze.html) diagrams if a query was executed via the [vectorized execution engine](../v20.2/vectorized-execution.html). This has been removed due to being confusing and probably unhelpful. [#53153][#53153]
+- `stall time` has been renamed to `IO time` in [`EXPLAIN ANALYZE`](../v20.2/explain-analyze.html) diagrams for queries executed via the [vectorized execution engine](../v20.2/vectorized-execution.html). [#53153][#53153]
+- Implemented geometry [builtin](../v20.2/functions-and-operators.html#built-in-functions) `ST_RemovePoint` [#53244][#53244]
+- An [`INSERT ... ON CONFLICT DO UPDATE`](../v20.2/insert.html#on-conflict-clause) statement without a list of column names after `ON CONFLICT` now results in a SQL syntax error with the error code `42601`. Previously, it errored with the message "there is no unqiue or exlcusion constraint matching the ON CONFLICT specification" and the error code `42P10`. [#53067][#53067]
+- Added support for the `WITH DATA` and `WITH NO DATA` arguments for the `REFRESH MATERIALIZED VIEW` command. [#53052][#53052]
+- Added support for the `GRANT ... ON SCHEMA command`. [#53344][#53344]
+- Added the `SHOW TYPES` command to list all user-defined types. [#53386][#53386]
+- Implemented the `ST_IsClosed`, `ST_IsSimple`, and `ST_IsRing` [builtins](../v20.2/functions-and-operators.html#built-in-functions). [#53240][#53240]
+
+### Command-line changes
+
+- The --storage-engine CLI flag no longer defaults to using the last-used storage engine when unspecified. Instead, pebble is used by default unless otherwise specified using the --storage-engine flag. [#52944][#52944]
+- Adds alias commands `ls` and `rm` for `userfile list` and `userfile delete`. [#53074][#53074]
+- The new debug command `decode-proto` reads descriptor from stdin in hex or base64 format (auto-detected) and a flag --schema=<fully qualified name to decode> with default value `cockroach.sql.sqlbase.Descriptor` and outputs to stdout the deserialized proto in JSON format. If the input is not a hex/base64 encoded proto then it is outputted verbatim. [#52972][#52972]
```

Comment:
nit: --schema=<fully qualified name to decode> >`--schema=<fully qualified name to decode>`